### DISCUSSION TITLE
HTTP协议及安全防范 | hengshuai's blog

### DISCUSSION THREAD

**Discussion by giscus** [ihengshuai/blog#97] [Announcements]
# HTTP协议及安全防范 | hengshuai's blog

HTTP超文本传输协议是一个用于 Web 应用程序通信的应用层协议。它是一种客户端-服务器协议，客户端通过发送请求到服务器来获取资源，服务器则根据请求返回响应。HTTP协议也是受攻击的重灾区如XSS、CSRF攻击，学会使用相关措施如CSP、HTTP安全防范等有效防止攻击发生

https://blog.usword.cn/frontend/computer-network/http-protocol.html

**Comment by ihengshuai** [comment]
HTTP协议万字长文，期待每位读者的点评纠正
